source/configuration/write.txt

```diff
@@ -31,40 +31,57 @@ The following options for writing to MongoDB are available:
    * - Property name
      - Description
 
-   * - ``mongoClientFactory``
-     - | MongoClientFactory configuration key.
-       | You can specify a custom implementation which must implement the
-         ``com.mongodb.spark.sql.connector.connection.MongoClientFactory``
-         interface.
-       |
-       | **Default:** ``com.mongodb.spark.sql.connector.connection.DefaultMongoClientFactory``
+   * - ``collection``
+     - | **Required.**
+       | The collection name configuration.
 
    * - ``connection.uri``
      - | **Required.**
        | The connection string configuration key.
        |
        | **Default:** ``mongodb://localhost:27017/``
 
+   * - ``convertJson``
+     - | When ``true``, the connector parses the string and converts extended JSON
+         into BSON.
+       |
+       | **Default:** ``false``
+
    * - ``database``
      - | **Required.**
        | The database name configuration.
 
-   * - ``collection``
-     - | **Required.**
-       | The collection name configuration.
+   * - ``idFieldList``
+     - | Field or list of fields by which to split the collection data. To
+         specify more than one field, separate them using a comma as shown
+         in the following example:
+
+       .. code-block:: none
+          :copyable: false
+
+          "fieldName1,fieldName2"
+
+       | **Default:** ``_id``
 
+   * - ``ignoreNullValues``
+     - | When ``true``, the connector ignores any ``null`` values when writing,
+         including ``null`` values in arrays and subdocuments.
```

Comment:
Suggestion:
Rather than "subdocuments", I think it would be good to use the terminology "embedded" or "nested" documents as identified on https://www.mongodb.com/docs/manual/tutorial/query-embedded-documents/